android/KMAPro/kMAPro/src/main/res/values/strings.xml

```diff
@@ -135,6 +135,9 @@
   <!-- Context: Keyman Settings menu / Spacebar caption hint -->
   <string name="spacebar_caption_hint_blank" comment="Spacebar caption hint - blank">Do not show caption on spacebar</string>
 
+  <!-- Context: Keyman Settings menu / Haptic feedback -->
+  <string name="haptic_feedback" comment="haptic feedback on key press">Vibrate when typing</string>
```

Comment:
It's fine for the internal name to be `haptic_feedback` and the user-visible text to be "Vibrate when typing" :grin: